packages/tools/devtools/devtools-browser-extension/e2e-tests/tsconfig.json

```
@@ -1,11 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../../common/build/build-common/tsconfig.test.node16.json",
 	"compilerOptions": {
-		"module": "esnext",
-		"outDir": "../dist/e2e-tests",
-		"composite": true,
+		"rootDir": "./",
+		"outDir": "../../lib/e2e-tests",
```

No changes to relative folder structure, right?
```suggestion
		"outDir": "../lib/e2e-tests",
```